1-mandatory/2-people-i-know.js

```diff
@@ -393,16 +396,33 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+
+let friendsWhoAreColleaguesOfStacie = friends.filter(
+  person => person.colleagues.some( colleague => colleague.name === "Stacie Villarreal"))
+  .map( person => `${person.name.first} ${person.name.last}`)
+
+  // console.log(friendsWhoAreColleaguesOfStacie)
 /*
 5) Find "Multi-tasking" colleagues
 Next, I want you to find all of the colleagues of my friends who are good at "Multi-tasking"
 You can tell if they are good at "Multi-tasking" because they will have it listed in their skills
 This time, I only want the full names of the people who can multitask
 */
 
-let colleaguesWhoCanMultitask = [];
+// let colleaguesWhoCanMultitask = friends.filter( person => person.colleagues.some( colleague => colleague.skills.includes("Multi-tasking")))
+// .map( person => 
+//   person.colleagues.name);
+  let colleaguesWhoCanMultitask = []
+
+  for(let person of friends){
+    for(let colleague of person.colleagues){
+      if(colleague.skills.includes("Multi-tasking")){
+        colleaguesWhoCanMultitask.push(colleague.name)
```

Comment:
Do you need to maintain the unquieness of `colleaguesWhoCanMultitask`? Is there any deuplicated colleague name?